packages/react-components/src/alert/src/Alert.tsx

```diff
@@ -60,7 +60,7 @@ export interface InnerAlertProps {
      * @param {SyntheticEvent} event - React's original SyntheticEvent.
      * @returns {void}
      */
-    onDismiss?: (event: SyntheticEvent) => void;
```

Comment:
not very useful, could still be synthetic event

Reply:
What do you mean by not useful?

Reply:
This was meant as a note to myself, but the idea would be that even if onDismiss is passed to a button onClick handler, onDismiss is not necessarely tied to a MouseEvent in our "public contract" which is the interface.

 For instance if we had an alert that would be dismissed after x seconds, the event would not be tied to a MouseEvent, but onDismiss could still be called